controllers/controllers_test.go

```diff
@@ -117,6 +141,8 @@ var _ = Describe("Testing HelmChartProxy and HelmReleaseProxy reconcile", func()
 		cluster := cluster1.DeepCopy()
 		err := k8sClient.Create(ctx, cluster)
 		Expect(err).ToNot(HaveOccurred())
+		err = k8sClient.Create(ctx, newKubeconfigSecretForCluster(cluster))
```

Comment:
```suggestion
		Expect(k8sClient.Create(ctx, newKubeconfigSecretForCluster(cluster))).To(Succeed())
```

I think this style is more readable, but your change is parallel with the rest of the tests, so 🤷🏻.